xml/dhcp_vendor_class.xml

```diff
@@ -203,4 +213,94 @@
     <param pos="0" name="os.family" value="Windows"/>
   </fingerprint>
 
-</fingerprints>
\ No newline at end of file
+  <fingerprint pattern="^(?i)Cisco Systems Inc\. Wireless Phone (\d{4}g?)$" certainty="0.8">
+    <description>Cisco Wireless Phone</description>
+    <example hw.model="7920">Cisco Systems Inc. Wireless Phone 7920</example>
+    <example hw.model="7925G">Cisco Systems Inc. Wireless Phone 7925G</example>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="hw.product" value="Unified Wireless IP Phone {hw.model}"/>
+    <param pos="0" name="os.vendor" value="Cisco"/>
+    <param pos="0" name="os.product" value="Unified Wireless IP Phone {hw.model} Firmware"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Cisco systems, Inc\. IP Phone CP(39\d{2})$" certainty="0.8">
+    <description>Cisco SIP Phone</description>
+    <example hw.model="3911">Cisco systems, Inc. IP Phone CP3911</example>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="0" name="hw.vendor" value="Cisco"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="hw.product" value="Unified SIP Phone {hw.model}"/>
+    <param pos="0" name="os.vendor" value="Cisco"/>
+    <param pos="0" name="os.product" value="Unified SIP Phone {hw.model} Firmware"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Cisco Systems, Inc\. IP Phone CP-(\d{4}(?:G|G-GE)?)$" certainty="0.8">
```

Comment:
I mentioned this on our call today, but wanted to document in the PR. We should consider matching the models that include `nr` after the numeric model number (there are 4 `cisco:unified_ip_phone_` CPEs that include the model number).